__tests__/api.test.js

```diff
@@ -164,4 +165,28 @@ describe("API tests", () => {
             })
          })
     })
+    describe("GET /api/users", () => {
+        test("200: Return an array of all users", () => {
+            return request(app)
+            .get("/api/users")
+            .expect(200)
+            .then(({body}) => {
+                expect(Array.isArray(body)).toBe(true)
+                expect(body.length).toBeGreaterThan(0)
+                body.forEach((user) => {
+                    expect(user).toHaveProperty("username")
+                    expect(user).toHaveProperty("name")
+                    expect(user).toHaveProperty("avatar_url")
+                })
+            })
+        })
+        test("404: Responds with error if path not found", () => {
+            return request(app)
+            .get("/api/notHere")
+            .expect(404)
+            .then((response) => {
+            expect(response.body).toEqual({msg: "Not found"})
+            }) 
+        })
```

Comment:
This is a generic 404 test and has nothing to do with the endpoint /api/users so it should not be within this describe block, but it is a good test to see.